submissions/tosinKolawole/HTML/index.html

```diff
@@ -0,0 +1,58 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset=@"UTF-8">
+    <title>Infinix Note series</title>
+</head>
+<body>
+      <header>
+            <h1><b>INFINIX NOTE 5</b></h1>
+            <h4>The most amazing phone yet in the Infinix note series.</h4>
+       </header>
+
+      <figure>
+        <img src="https://images-na.ssl-images-amazon.com/images/I/61JH1324jTL._SL1040_.jpg">
+        <img src="https://www.gizmochina.com/wp-content/uploads/2018/06/Infinix-Note-5-600x562.jpg">
+      </figure>
+
+     <!--This section is for product price-->
+      <section>
+          <h2><b>Product Price<b></h2>
+          <article>
+             <p> 
+              The Infinix Note 5 Price in Nigeria is best on Beta Market Online stores and its current price starts at N57,000 naira for the 3GB variety and N65,000 naira for the 4GB variety.    
+             </p>
+ 
+     <!-- This section is for product description -->
+     <section>
+        <h2><b>Product Description</b></h2>
+        <article>
+            <p>
+            Infinix Note 5 smartphone runs on Android v8.1 (Oreo) operating system. The phone is powered by Octa core (2.3 GHz, Quad core, Cortex A53 + 1.65 GHz, Quad core, Cortex A53) processor. It runs on the MediaTek MT6763T Chipset. It has 3 GB RAM and 32 GB internal storage.
+           </p>
+           <p>
+            Infinix Note 5 smartphone has a IPS LCD display. It measures 158 mm x 75 mm x 8.4 mm and weighs 175 grams. The screen has a resolution of 1080 x 2160 pixels and 403 ppi pixel density. It has an aspect ratio of 18:9 and screen-to-body ratio of 78.14 %. On camera front, the buyers get a 16 MP f/2.0 Primary Camera(1µm pixel size) and on the rear, there's an 12 MP camera with features like Fixed Focus. It is backed by a 4500 mAh battery. Connectivity features in the smartphone include WiFi, Bluetooth, GPS, Volte, and more.
+           </p> 
+
+       </article>
+    </section>
+    
+    <!--This section is for Similar products -->
+    <section>
+         <h2>Similar Product</h2>
+         <div>
+           <ol>
+                 <li><strong> Infinix Note 5 Stylus</strong></li>
+                 <li><strong> Infinix Hot 7</strong></li>
+                 <li><strong> Infinix Note 4 Pro</strong></li>
+           </ol>
+           </div> 
+     </section>    
+         
+   <section>
+          <button>Buy Now</button>      
+   </section>   
+
```

Comment:
Remove whitespace.

Reply:
I really don't know how to do that yet.. I checked Google and the answers I got are confusing..can you kindly explain?

Reply:
I mean the empty line